tests/test_app.py

```diff
@@ -158,19 +159,18 @@ def test_it_works_with_multiline_in_patch_p(self) -> None:
                             | path: creds.py
                             | value: 'V84C7sDU001tFFodKU95USNy97TkqXymnvsFmYhQ'
                             | entropy_validation: BASE64STDPAD_CHARS 4.784184 True]
-                        | ml_validation: NOT_AVAILABLE
                     rule: Token
                         | severity: medium
                         | confidence: moderate
+                        | ml_probability: 0.9982267618179321\n
```

Comment:
```suggestion
                        | ml_probability: 0.9982267618179321
```

`\n` line escape seems can be removed. Please check.